tests/unscheduled/eip7928_block-level_access_lists/spec.py

```diff
@@ -0,0 +1,44 @@
+"""Reference spec for [EIP-7928: Block-level Access Lists.](https://eips.ethereum.org/EIPS/eip-7928)."""
```

Comment:
Thanks @raxhvl. Looks like we need to rename without the hypen to keep mypy happy and make this module a valid package name :)
-> `tests/unscheduled/eip7928_block_level_access_lists/spec.py`

Reply:
Ty @danceratopz. I placed the test under `unscheduled`. @marioevz suggested to created a top level folder for EIP under `tests` but that sounded like a _rude intervention_ to me :P so i tucked it away.

My goal with this PR is to involve you guys early so we don't have to review a mega PR. I have added bareshell tests and I'll create sub PRs to fill these in small batches. I will also use this PR for high level discussions.